src/UI/examples/Icon/Standard/disabled_icon.php

```diff
@@ -7,17 +7,11 @@ function disabled_icon()
 
 	$buffer = array();
 
-	$ico = $f->icon()->standard('crs', 'Course', 'small', false);
+	$ico = $f->icon()->standard('grp', 'Course', 'large', false)->withDisabled(true);
 
-	$buffer[] = $renderer->render($ico->withDisabled(true))
-		.' Small Course';
-
-	$buffer[] = $renderer->render($ico->withSize('medium')->withDisabled(true))
-		.' Medium Course';
-
-	$buffer[] = $renderer->render($ico->withSize('large')->withDisabled(true))
-		.' Large Course';
```

Comment:
Why did you remove the other sizes?

Reply:
Because this example is about disabling, not about sizes. We have another one for that. To me it is more important to show disabling for outlined and non-outlined than various sizes. If we do both, we have an overload here.